### ISSUE TITLE
Source for 1.2.3 won't build as released

### ISSUE THREAD

**Issue by Eupeptic1** [Simon-Tesla/RaccoonyWebEx#62]
## tl;dr
When building, gulp complains about "task never defined".  Moving what is currently [line 101 of gulpfile.js](https://github.com/Simon-Tesla/RaccoonyWebEx/blob/4efd6a337272d11ae7e54a162dd615ea0315d7a5/gulpfile.js#L101) to *after* the definition of the sign.firefox task (currently lines 103-119) fixes it.

## Long version:
The released version of 1.2.3, as downloaded in a zip file from Github, won't build as-is for me.

## Steps to reproduce:
1.  Go to https://github.com/Simon-Tesla/RaccoonyWebEx , click the green "Clone or download" button, and select the "Download ZIP" option.

2.  Unpack the downloaded zip file to a new, empty directory on the local filesystem.

3.  Run the first two steps described in README.md (`npm install -g typescript gulp` and `npm install`).

4.  Run the `gulp build` step.

Expected result: Add-on builds; `dist/` is populated with the built files.

Actual result: gulp fails on an assertion:

```shell
$ gulp build

/home/eupeptic/src/webex-rac/orig-1.2.3/node_modules/undertaker/lib/helpers/normalizeArgs.js:15
    assert(fn, 'Task never defined: ' + task);
    ^
AssertionError [ERR_ASSERTION]: Task never defined: sign:firefox
    at getFunction (/home/eupeptic/src/webex-rac/orig-1.2.3/node_modules/undertaker/lib/helpers/normalizeArgs.js:15:5)
    at map (/home/eupeptic/src/webex-rac/orig-1.2.3/node_modules/arr-map/index.js:20:14)
    at normalizeArgs (/home/eupeptic/src/webex-rac/orig-1.2.3/node_modules/undertaker/lib/helpers/normalizeArgs.js:22:10)
    at Gulp.series (/home/eupeptic/src/webex-rac/orig-1.2.3/node_modules/undertaker/lib/series.js:13:14)
    at Object.<anonymous> (/home/eupeptic/src/webex-rac/orig-1.2.3/gulpfile.js:101:24)
    at Module._compile (internal/modules/cjs/loader.js:1156:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1176:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Module.require (internal/modules/cjs/loader.js:1042:19)
```
## Versions and environment

nodejs 12.16.2, npm 6.14.4, gulp CLI 2.2.0 and local 4.0.2 .

Linux desktop (Debian 9), amd64/x86_64.

## Cause and possible fix

The gulp task for `sign` is defined first in `gulpfile.js`.  It refers to a `sign.firefox` task, which is defined later, which I *think* is causing the problem.

Moving the definition of the `sign` task to *after* `sign.firefox` fixes it for me; gulp can build the extension as normal.

edit: formatting

**Comment by Simon-Tesla** [comment]
Whoops; thought for sure I tested the build before checking that change in, but apparently not.
